raiden/tests/fixtures/blockchain.py

```diff
@@ -90,9 +90,10 @@ def pregenerate_dag(request, blockchain_type, dagpath):
     )
 
     if blockchain_type == 'geth' and missing_dag:
-        os.makedirs(os.path.dirname(dagpath))
+        dag_folder = os.path.split(dagpath)[0]
```

Comment:
yeah sure no problem with the variable name, just don't end up getting the parent's parent